pkg/common/types/sandbox.go

```diff
@@ -5,18 +5,17 @@ import (
 	"time"
 )
 
-type SandboxRedis struct {
+type SandboxStore struct {
```

Comment:
SandboxStore  looks like a class, while here we want to express a object
Maybe call `SandboxInfo`